src/config.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "elastic",
+    "type": "executer",
+    "imgUrl": "logo.png",
+    "execProgram": "node",
+    "main": "app.js",
+    "version": "1.0.0",
+    "description": "Integrate to Elastic NodeJS client",
+    "settings": [
+      {
+        "name": "connetionString",
+        "viewName": "Connection String",
+        "type": "string"
+      }
+    ],
+    "methods": [
+      {
+        "name": "INDEX_DATA",
+        "viewName": "Index Elastic",
+        "params": [
+          {
+            "name":"obj",
+            "viewName": "Object to write",
+            "type": "string"
+          }
+        ]
+      },
+      {
+        "name": "SEARCH_DATA",
+        "viewName": "Search Elastic",
```

Comment:
* No need to add the "Elastic" word to every method. This is an elastic plugin, it is obvious.
* The name for the param needs to be "Search Object"